### ISSUE TITLE
Specifying strand-specific sequencing?

### ISSUE THREAD

**Issue by acesnik** [jtpoirier/proteomegenerator#5]
It looks like you used an unstranded RNA-Seq protocol in your recent paper. You might want to add a setting to be able to specify strand-specific sequencing in the configuration, so that type of data can be analyzed properly.

Cheers,

AC

**Comment by jtpoirier** [comment]
Yes, this is a good suggestion and something we've thought about. Happy to accept pull requests if you want to add this in the mean time.
